catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -443,4 +443,12 @@
         <bundle>mvn:org.apache.mina/mina-core/${mina.version}</bundle>
     </feature>
 
+    <feature name="catalog-async-inmemory" install="auto" version="${project.version}">
```

Comment:
I would like to move towards not enabling experiment APIs by default too quickly. My preference would be for it to stay off by default for now, especially with 2.10 coming up.